.github/workflows/scala-steward.yml

```diff
@@ -0,0 +1,16 @@
+# This workflow will launch at 09:00 every Monday
+on:
+  schedule:
+    - cron: '0 9 * * 1'
+  workflow_dispatch:
```

Comment:
```suggestion
```

Reply:
According to the docs, this allows to trigger Scala Steward manually. I wanted to use this to verify if the username is indeed `moia-pipeline`.

Reply:
Oh wow, an empty config does that? 🙈

Reply:
https://github.com/scala-steward-org/scala-steward-action#how-can-i-trigger-a-run